packages/rune-games-cli/src/flows/Upload/ReadyForReleaseStep.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect, useMemo } from "react"
+
+import { Choose } from "../../components/Choose.js"
+import { Step } from "../../components/Step.js"
+
+export function ReadyForReleaseStep({
+  onComplete,
+}: {
+  onComplete: (readyForRelease: boolean) => void
+}) {
+  const [readyForRelease, setReadyForRelease] = useState<boolean | undefined>()
+
+  useEffect(() => {
+    if (readyForRelease !== undefined) onComplete(readyForRelease)
+  }, [readyForRelease, onComplete])
+
+  const label = useMemo(() => {
+    if (readyForRelease === undefined) return "Is your game ready for release?"
+
+    return readyForRelease
+      ? "Game is ready for release"
+      : "Game is not ready for release"
```

Comment:
```suggestion
    return readyForRelease
      ? "Yes, please review and release it"
      : "No, just uploading for playtesting"
```

Reply:
Sounds good. Just note that the part you suggest to change is what will be displayed **after** dev already made the choice.

During the choice we still display `Is your game ready for release?` with `No`/`Yes`

Reply:
I see, thanks. We could use the text you suggested here and the text I suggested for the “options” variable below.

Reply:
After discussion with @bfelbo, having so long answers and my previous suggestion may be confusing for devs hence changing only info after the question is answered (https://github.com/rune/rune-games-sdk/pull/185/commits/4d633acf5ec8770c43ec488e7507b752057c5b89)